sycl/test-e2e/CMakeLists.txt

```diff
@@ -36,6 +36,12 @@ find_package(Python3 REQUIRED COMPONENTS Interpreter)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/lit.site.cfg.py.in"
                "${CMAKE_CURRENT_BINARY_DIR}/lit.site.cfg")
 
+if(NOT SYCL_TEST_E2E_STANDALONE)
+  list(APPEND SYCL_E2E_TEST_DEPS
+    sycl-toolchain
```

Comment:
Strictly speaking, there are other test utilities deps like `FileCheck`, but should normally be an issue on the very first invocation only, and we can address it later.

Reply:
I can add `FileCheck` and maybe also `not` if you want, but, as you say, we can address that in a different one.

Reply:
No, I think we're good to call @intel/llvm-gatekeepers to merge this one.